agent/action/pipeline_manager.py

```
@@ -232,16 +232,23 @@ def _ensure_cache_loaded(force_refresh=False):
 
     # fallback：路径结构调整时的保底
-    if not target_path.exists():
-        _fallback = _project_root / "resource" / "pipeline"
-        if _fallback.exists():
-            target_path = _fallback
-        else:
-            # 优先级列表：父目录名越靠前越优先；同时过滤掉不含 json 的空文件夹
+    if not target_path.exists() or not any(target_path.rglob("*.json")):
+        _fallback_candidates = [
+            _project_root / "resource" / "base" / "pipeline",  # 当前主流部署结构
+            _project_root / "resource" / "pipeline",           # 旧版结构（注意：更新器可能残留空目录）
```

Ruff (0.15.10)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)